resources/js/components/global/Dashboard.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+  <div class="test">
+    <DashboardNotification></DashboardNotification>
+    <div class="content mt-5">
+      <div class="row justify-content-center" v-for="(item, index) in headings" :key="index">
```

Comment:
Probably should just use a simple <ul> here (maybe with a class of ".text-align-center") instead of multiple rows